seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -1440,7 +1440,7 @@
       "id": "bad87fee1348bd9aedf08830",
       "title": "Add Placeholder Text to a Text Field",
       "description": [
-        "Your placeholder text is what appears in your text <code>input</code> before your user has input anything.",
+        "Placeholder text is what is displayed in your <code>input</code> element before your user has input anything.",
```

Comment:
This should be past-tense (and yes, it was incorrect before as well): "before your user has inputted anything."